dashboard/test/models/user_test.rb

```diff
@@ -1455,6 +1455,25 @@ def create_level_group(sub_level_name)
     assert user.under_13?
   end
 
+  test 'over 21' do
+    user = create :user
```

Comment:
Mostly reused the testing logic for under 13. The caveat: new users created have a default age of 21+, so we do not refute the first case of 'create :user'. Instead, we set their age and then refute.